### ISSUE TITLE
[Feature] File explorer

### ISSUE THREAD

**Issue by cualquiercosa327** [LightningMods/Itemzflow#18]
Hi, first thanks for this wonderful program.
I would love suggest if it can be add a file explorer (or maybe launch lappyfile Explorer) when
the game/fpkg is mounted which allows delete, copy, cut.. files of the game to /data/folder and viceversa. 
Thanks!

**Comment by LightningMods** [comment]
Yes, I've been thinking of extending the already implemented file selector to include options such as copy, paste etc as well as adding a dedicated file manager option

**Comment by z0mb3** [comment]
A included full "file explorer" feature really would make things easier.
Especially taking a look on https://www.reddit.com/r/PS5_Jailbreak/comments/1k4popi/all_ps5_games_from_internal/
which describes how to use backups from the internal m2 drive (/mnt/ext1)
By using the already available moving functions and having a look on the already prepared two folder sided UI - you might get the impression everything is already there and internally in use by the dev :D

As well Itemzflow seems to have more privilieges on the /mnt/ext1 than lappys file explorer.
